acceptancetests/utility.py

```diff
@@ -247,8 +247,8 @@ def add_arg_juju_bin(parser):
                         default=None)
 
 
-def add_basic_testing_arguments(parser, using_jes=False, deadline=True,
-                                env=True):
+def add_basic_testing_arguments(
+        parser, using_jes=False, deadline=True, env=True, existing=False):
```

Comment:
As mentioned, consider making this with a default of false in the argument, and drop from the method.